fivejars/openy_gated_content
# Shared content stat

Steps for testing:
1. - [ ] Enable shared content server module at one of the builds. Make all content available to share at `admin/content` page
2. - [ ] Add the first server as a source server instead of shared.openy at: `admin/virtual-y/shared-content/server/add`
3. - [ ] Open fetch form at the second server and try to download content to your local Virtual Y
4. - [ ] Verify that count of downloads increased after successful download at the first server reporting page: `/admin/virtyal-y/shared-content/download-stat`
5. - [ ] Verify that count of downloads updated at the fetch form.
6. - [ ] Check filters and the ability to export in csv at the first server reporting page.
